doc/install.xml

```diff
@@ -8,14 +8,24 @@ This is the chapter of the documentation describing the installation.
 <Heading>Installation of the <Package>recog</Package> package</Heading>
 <Index><Package>recog</Package></Index>
 
-To install this  package, just extract the package's  archive file to
-the  GAP  <F>pkg</F>  directory.<P/>
+To install this package, just extract the package's archive file to the &GAP;
+<F>pkg</F> directory.<P/>
 
-By default, the <Package>recog</Package> package is not automatically
-loaded by &GAP; when it is installed. You must load the package with
+By default, the <Package>recog</Package> package is not automatically loaded
+by &GAP; when it is installed. You must load the package with
```

Comment:
That actually doesn't seem to be accurate anymore, at least for me, it does get loaded because `atlasrep` 2.x suggests it... I am not exactly about that, though, but that's how it is.

Reply:
I've changed it to something a bit more general.